sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -24,8 +24,10 @@
 
 
 from typing import Any, Dict, List, Optional, Sequence, Union, Tuple, Mapping, Type, cast
+from typing_extensions import Literal
 import warnings
```

Comment:
Stick `import warnings` on line 25 to make pylint happy (it doesn't like 3rd party imports placed above 1st party imports)